libraries/AP_HAL_ChibiOS/hwdef/AEDROXH7/README.md

```diff
@@ -0,0 +1,93 @@
+# AEDROX H7 Flight Controller
+
+
+## Features
+
+- STM32H743 microcontroller
+- ICM42688-P IMU with external clock
+- DPS368 barometer
+- 10V 2.3A BEC; 5V 2.3A BEC
+- Flash Memory
+- 7x UART
+- 8x PWM
+- 1x I2C
+- 1x SPI
+- 1x SWD
+
+## Pinout
+
+![AEDROX H743](AEDROX_FC_H7_PINOUT.jpg "AEDROX H743")
+
+## UART Mapping
+
+ - SERIAL0 -> USB
+ - SERIAL1 -> UART1 (MAVLink2, DMA-enabled)
+ - SERIAL2 -> UART2 (GPS, DMA-enabled)
+ - SERIAL3 -> UART3 (RCIN, DMA-enabled) 
+ - SERIAL4 -> UART4 (MAVLink2, DMA-enabled)
+ - SERIAL5 -> UART7 (ESC Telemetry, DMA-enabled)
+ - SERIAL6 -> UART8 (MAVLink2, DMA-enabled)
+
+## RC Input
+
+The default RC input is configured on the UART3 (SBUS). Non SBUS,  single wire serial inputs can be directly tied to RX3 if SBUS pin is left unconnected. RC could  be applied instead at a different UART port such as UART1, UART4 or UART8, and set the protocol to receive RC data: `SERIALn_PROTOCOL=23` and change SERIAL3 _Protocol to something other than '23'.
+
+
+## VTX Support
+
+The SH1.0-6P connector supports a DJI Air Unit / HD VTX connection. Protocol defaults to DisplayPort. Pin 1 of the connector is 10v so be careful not to connect this to a peripheral requiring 5v.
```

Comment:
add info on VTX sw GPIO83 and note that relay3 is setup by default
also that DisplayPort OSD is enabled by default via  SERIAL8